.github/workflows/pr-unit-tests.yml

```diff
@@ -10,17 +10,17 @@ jobs:
       fail-fast: false
       matrix:
         # We test against the earliest and latest PHP versions for each major supported version.
-        php: [ '7.1', '7.4', '8.0', '8.3' ]
+        php: [ '7.2', '7.4', '8.0', '8.3' ]
```

Comment:
Not necessarily something to address in this PR 🙌🏼 ... but at some point we probably should replace `'8.3'` with `'8.4'`.